### PR TITLE
change to semiautocuts for chi2 cut

### PR DESCRIPTION
**Pull request by michrw**
Changes to the elif statement on line 624. Without the change it will crash sense self.cut_pars and cut_pars are not always the same. 
`        elif ('val_lower' in self.cut_pars):
            self.values_lower[on_cut_bin-1] = self.cut_pars['val_lower']
            value_lower = self.cut_pars['val_lower']
            cut_mask = self.df[self.cut_rq].values > self.cut_pars['val_lower']`

### CODE REVIEW

File: detanalysis/semiautocuts.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def __init__(self, vaex_dataframe, cut_rq, channel_name,
                  exceptions_dict={},
                  ofamp_rq=None, chi2_rq=None,
                  cut_name=None, cut_rq_name_override=False,
-                 lgc_diagnostics=False):
```
Comment (rkromani) [inline detanalysis/semiautocuts.py]: I'd prefer to keep these defaults as false, so we don't print out messy things by default... as an alternative, you could have the diagnostics for the functions below turned on and off globally by a variable stored as part of the objects?

File: detanalysis/semiautocuts.py [hunk 2]
```diff
@@ -2203,5 +2202,11 @@ def get_randoms_passfrac_table(self):
         print(tabulate(tab_table, headers = headers_, tablefmt = 'orgtbl', floatfmt='.3f'))
 
 
+    def get_final_values(self,feature_exp):
```
Comment (rkromani) [inline detanalysis/semiautocuts.py]: this is the same as calling df.cut_blah.values... is it worth having as a function here?

Comment (michrw) [reply detanalysis/semiautocuts.py]: Ok honestly wasn't sure if that would work. If you think there is already functionality for this then we don't need the new function for it

File: detanalysis/semiautocuts.py (hunk 1 above)
Comment (michrw) [reply detanalysis/semiautocuts.py]: Sorry, I forgot to turn these off!

**Review by rkromani — APPROVED** [review]

**Pull request merged**